src/Metadata/Property/PropertyMetadata.php

```diff
@@ -43,6 +44,7 @@ public function __construct(Type $type = null, string $description = null, bool
         $this->identifier = $identifier;
         $this->iri = $iri;
         $this->attributes = $attributes;
+        $this->childInherited = null;
```

Comment:
If you see a reason to add this to the constructor I'm adding it but I don't see one as it should always be filled by `InheritedPropertyMetadataFactory`, if it has properties from a child class.